--- a/setup.py
+++ b/setup.py
@@ -4,7 +4,7 @@
 import zulip
 
 import os
-from distutils.core import setup
+from setuptools import setup
 
 def recur_expand(target_root, dir):
   for root, _, files in os.walk(dir):
@@ -32,4 +32,5 @@
                                              "examples/print-messages"])] + \
           list(recur_expand('share/zulip', 'integrations/')),
       scripts=["bin/zulip-send"],
+      install_requires=['simplejson>=2.3.2', 'requests>=0.12.1'],
      )
